DayrSuryan/DSEthiop7.xml

```diff
@@ -349,10 +412,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      The manuscript could be copied
                      by <persName ref="PRS11959Pawlos" role="scribe"/>,
                      in <placeName ref="LOC2804Egypt">Egypt</placeName> or in <placeName ref="LOC2991Eritre">Eritrea</placeName>.
-                     A brief note identifies <persName ref="PRS12557Barha">ḫalaqā Barha</persName> as the owner.
+                     A brief note identifies <persName ref="PRS12557Barha">ḫalaqā Barha</persName>, who is also mentioned in
```

Comment:
"The colophon identified ... Barha as the owner. According to a ... added probably later, in pencil, in a secondary hand, the scribe was Pawlos who copied the book in .... and owned it." (I think Eritrea should stand first, Egypt is less probable).  If we do not consider the note on Pawlos very credible (added later, in a secondary hand), then we should not use its date 1927/28, but rather say smth like "first half of the 20th century".